registration.php

```diff
@@ -14,518 +35,621 @@
     'Nagaland', 'Orissa', 'Pondicherry', 'Punjab', 'Rajasthan', 'Sikkim', 'Tamil Nadu', 'Tripura',
     'Uttaranchal', 'Uttar Pradesh', 'West Bengal');
 
-// Validate Post Data and insert Record
 if (!empty($_POST))
 {
     // Include validate file
-    include('helper/validate.php');
+    require_once('helper/validate.php');
 
     if (!$error)
```

Comment:
Here validate.php is included in above line and $error is checked from that file.
such kind of code may lead to a confusion when you includes many files and checking the variable from that file here.
it will be difficult to find that variable is defined in which file and where is the issue.
so
better to write codes of helper/validate.php as simple function or oops concept what ever you think is better and then call the methods by passing required inputs and then return the required outputs and store in a variable and use.
something like
$is_error = validate();